regression/run.ts

```diff
@@ -401,87 +401,95 @@ async function generateDiff(
   jsonTemplate: string
 ): Promise<void> {
   process.stdout.write('  - Comparing output ');
-  const repoSUSHIDir1 = config.getRepoSUSHIDir(repo, 1);
-  const repoSUSHIDir2 = config.getRepoSUSHIDir(repo, 2);
-  const [v1Files, v2Files] = await Promise.all([
-    getFilesRecursive(repoSUSHIDir1),
-    getFilesRecursive(repoSUSHIDir2)
-  ]);
-  const files = union(
-    v1Files.map(f => path.relative(repoSUSHIDir1, f)),
-    v2Files.map(f => path.relative(repoSUSHIDir2, f))
-  );
-  files.sort();
-  let jsonResults = '';
-  //Don't use forEach because it can result in files out-of-order due to async
-  for (const file of files) {
-    const v1File = path.join(repoSUSHIDir1, file);
-    const v2File = path.join(repoSUSHIDir2, file);
-    const [v1Contents, v2Contents] = await Promise.all([readFile(v1File), readFile(v2File)]);
-    let v1Json: any = null;
-    let v2Json: any = null;
-    try {
-      v1Json = JSON.parse(v1Contents);
-      v2Json = JSON.parse(v2Contents);
-      if (isEqual(v1Json, v2Json)) {
+  try {
```

Comment:
Wow. GitHub really messes this diff up if you're showing whitespace changes. The only change is the surrounding try/catch. You can see this more easily if you tell it to hide whitespace in the diff: 
![CleanShot 2024-01-24 at 19 50 50@2x](https://github.com/FHIR/sushi/assets/2278253/dd18f1ed-ea16-4f64-9404-b1acb92fdd37)

Reply:
Holy gigantic image, Batman!